larq/conftest.py

```diff
@@ -19,6 +19,7 @@ def graph_mode():
     with context.graph_mode():
         with tf.compat.v1.Session().as_default():
             yield
+            tf.keras.backend.clear_session()
```

Comment:
Why is this necessary?

Reply:
This is required for TensorFlow < 2.1 see f43aa7b25495a6ede2f9139f10559dfd8e450713